packages/snap/src/polkadot/api.ts

```diff
@@ -1,20 +1,23 @@
-import {ApiPromise} from "@polkadot/api";
-import {WsProvider} from "@polkadot/api";
-import {getConfiguration} from "../configuration";
-import { SnapsGlobalObject } from "@metamask/snaps-types";
+import { getConfiguration } from '../configuration';
+import { ApiPromise, HttpProvider } from '@polkadot/api';
 
 let api: ApiPromise;
-let provider: WsProvider;
+let provider: HttpProvider;
```

Comment:
afaik it weorks a lot better with ws

Reply:
It is works better, but web sockets are not supported by metamask anymore